core/bessctl.cc

```diff
@@ -550,17 +573,26 @@ class BESSControlImpl final : public BESSControl::Service {
       });
     }
 
+    if (request->wid() < 0) {
```

Comment:
Isn't request->wid() already saved in wid_filter?

Also, is this if block meant to catch the case that no specific wid was requested?